### ISSUE TITLE
Jest plugin doesn't resolve the "rootPath" correctly

### ISSUE THREAD

**Issue by rdrezner** [webpro/knip#240]
Hello, thanks for creating and maintaining `knip` - I really like this tool.

When I scan my project, I'm getting some false positives and they're coming from the `jest` configuration. Files such as:
`<rootDir>/some_file.ts` are listed as unused. I have the `rootDir` property set to `./src` in `jest` config file.

I took a closer look at the `jest` plugin configuration and I noticed that in this line https://github.com/webpro/knip/blob/59d437d7184bbe5374da00be60ea25993eb63c72/src/plugins/jest/index.ts#L90 the plugin replaces the `<rootDir>/` string with the empty one. It seems to me that this is not correct for projects that have the `rootDir` property defined in the `jest` configuration (see: https://jestjs.io/docs/configuration#rootdir-string).

When I change this code line to: 
```
const replaceRootDir = (name: string) =>
    name.includes('<rootDir>') ? join(cwd, name.replace(/^.*<rootDir>/, config.rootDir ?? '')) : name;
```
knip doesn't list those file as unused which is the expected behaviour.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.25.0. See [Release 2.25.0](https://github.com/webpro/knip/releases/tag/2.25.0) for release notes._

**Comment by webpro** [comment]
Thanks for reporting this, @rdrezner!
